### ISSUE TITLE
error while loading shared libraries: libqt5keychain.so.1

### ISSUE THREAD

**Issue by steko** [flathub/org.qgis.qgis#160]
I believe the latest update to QGIS 3.22.3 ("stable" channel) caused the following error:

```
qgis: error while loading shared libraries: libqt5keychain.so.1: cannot open shared object file: No such file or directory
```

QGIS doesn't start at all.

**Comment by steko** [comment]
Oh I see this is probably addressed in e8a744bd85c048bb9177fb344812ef01117f284a already. I hope a fix is released soon then. Thanks!

**Comment by steko** [comment]
It's fixed in the current release 

```
flatpak info org.qgis.qgis/x86_64/stable

          ID: org.qgis.qgis
         Ref: app/org.qgis.qgis/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 3.22.3
     License: GPL-2.0+
      Origin: flathub
  Collection: org.flathub.Stable
Installation: system
   Installed: 2,1 GB
     Runtime: org.kde.Platform/x86_64/5.15-21.08
         Sdk: org.kde.Sdk/x86_64/5.15-21.08

      Commit: 5a02c99b5fd5adc52e559af7279fadab085ed81bcc6bfabd71f12c567dd82182
      Parent: 34971d9383a3b0bc6bdaaa5382ea6640664c648191c7e7dcfe82b06e5988c2b4
     Subject: Properly enable armadillo in gdal (f35733a8)
        Date: 2022-02-04 06:38:57 +0000
```
